docs/source/guides/contributor/chapters/environment.rst

```diff
@@ -3,14 +3,22 @@ Development environment
 
 .. include:: /.include/helpus.rst
 
-.. warning:: TODO: Needs improvment here. i.e: virtualenvs, GPG, etc.
+.. warning:: TODO: Needs improvement here. i.e: virtualenvs, GPG, etc.
 
 Installing dependencies
 -----------------------
 
 You need to install few dependencies before start coding::
 
- $ sudo dnf install gcc libvirt-devel
+ $ sudo dnf install gcc libvirt-devel python-devel
```

Comment:
Actually, libvirt-devel is not a requirement anymore because the "vm runner" is gone... and IIRC that's what required gcc and python-devel too.